jadx-core/src/main/java/jadx/core/xmlgen/ResTableParser.java

```diff
@@ -27,7 +28,7 @@
 public class ResTableParser extends CommonBinaryParser {
 	private static final Logger LOG = LoggerFactory.getLogger(ResTableParser.class);
 
-	private static final Pattern VALID_RES_KEY_PATTERN = Pattern.compile("\\$+[\\w\\d-_.]+");
+	private static final Pattern VALID_RES_KEY_PATTERN = Pattern.compile("[\\w\\d_]+");
```

Comment:
Btw, is there any chance for the `origKeyName` to have non-latin text? (ie. chinese/japanese/korean font)
Should i use `a-z` instead of `\w` ?